src/EnergyPlus/HVACUnitaryBypassVAV.cc

```diff
@@ -1849,6 +1849,13 @@ namespace HVACUnitaryBypassVAV {
             BypassDuctFlowFraction = 0.0;
         }
 
+        if (CBVAV(CBVAVNum).AirLoopNumber > 0 && DataLoopNode::Node(CBVAV(CBVAVNum).AirOutNode).MassFlowRate > 0.0) {
+            DataAirLoop::AirLoopFlow(CBVAV(CBVAVNum).AirLoopNumber).OAFrac = DataLoopNode::Node(CBVAV(CBVAVNum).MixerOutsideAirNode).MassFlowRate /
+                                                                             DataLoopNode::Node(CBVAV(CBVAVNum).AirOutNode).MassFlowRate;
+        } else {
+            DataAirLoop::AirLoopFlow(CBVAV(CBVAVNum).AirLoopNumber).OAFrac = 0.0;
+        }
+
```

Comment:
Testing showed that the the air loop OAFrac was not set a value for VAV bypass air terminal units. Thus, the value is set here. Needs more to test such that the latest ratio is used. Now this equation is set in init but this may have to move to calc function.